dm/dm/master/config.go

```diff
@@ -313,6 +314,10 @@ func (c *Config) adjust() error {
 		c.QuotaBackendBytes = quotaBackendBytesLowerBound
 	}
 
+	if c.ExperimentalFeatures.OpenAPI {
+		log.L().Warn("openapi is a GA feature and removed from experimental features, so this configuration will have no affect, please set openapi=true in dm-master config file")
```

Comment:
Should enable OpenAPI if it is upgraded from an old cluster

Reply:
but how should i do this in code ? do i need to update dm-master's config file for user ? 🙄

---

or what if user set

```toml
openapi = false

[experimental]
openapi = true
```

Reply:
what is the expected heavior when enable openapi  in v5.3 but forget to add new added config in dm-master after upgrade ? @sunzhaoyang  ptal

Reply:
after talked with pm, both `cfg.OpenAPI` and `s.cfg.ExperimentalFeatures.OpenAPI` will take effects when user want to enable openapi, added in https://github.com/pingcap/tiflow/pull/3836/commits/4c20104e25171fbb6d1ca81145283ccfde90e509

ptal @GMHDBJD

Reply:
> what is the expected heavior when enable openapi in v5.3 but forget to add new added config in dm-master after upgrade ? @sunzhaoyang ptal

Disabled. He has to enable manually.